arteria/models/runfolder_utils.py

```diff
@@ -0,0 +1,177 @@
+import os
```

Comment:
I think it's better if we only use pathlib, i.e.:
- use `Path.iterdir` instead of `os.listdir`
- use the the `/` operator to combine paths instead of `os.path.join` (e.g. check https://github.com/arteria-project/arteria/blob/5e0ebe1a29b10c54d86fa25f64323cc4e7018cba/arteria/models/runfolder_utils.py#L56)